src/SortableContainer/index.js

```diff
@@ -229,11 +285,17 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 			this.translate = translate[axis];
 
 			if (lockToContainerEdges) {
-				if (translate[axis] < this.minTranslate) {
-					translate[axis] = this.minTranslate;
-				} else if (translate[axis] > this.maxTranslate) {
-					translate[axis] = this.maxTranslate;
-				}
+				const [minLockOffset, maxLockOffset] = this.getLockPixelOffsets();
+				const minOffset = this.dimension / 2 - minLockOffset;
+				const maxOffset = this.dimension / 2 - maxLockOffset;
+
+				translate[axis] = Math.max(
```

Comment:
Do you prefer if replace it with a `limit(min, max, value)` or an `if .. else if`?